VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs

```diff
@@ -37,6 +38,32 @@ protected override void OnEnable()
 			_gate = target as GateAuthoring;
 		}
 
+		protected virtual void OnSceneGUI()
+		{
+			if (target is IEditableItemAuthoring editable) {
+				var position = editable.GetEditorPosition();
+				var transform = (target as MonoBehaviour).transform;
+				if (transform != null && transform.parent != null) {
+					position = transform.parent.TransformPoint(position);
+					var axis = transform.TransformDirection(Vector3.up);
+					var scale = _gate.Item.Data.Length * 0.0005f;

```

Comment:
That's world scale, right? If so, I'd use the global world scale constant we have (`VpxConverter.GlobalScale`).

Reply:
didn't know about this one i'll check that